spider/src/page.rs

```diff
@@ -39,20 +39,13 @@ impl Page {
     /// Find all href links and return them
     pub fn links(&self, domain: &str) -> Vec<String> {
         let mut urls: Vec<String> = Vec::new();
-        let selector = Selector::parse("a").unwrap();
-
+        let selector = Selector::parse(&format!(r#" a[href^="{}"] a[href$=".html"], a:not([href*="."]), a:not([href*=":"]) "#, domain)).unwrap();
```

Comment:
selects only current domains links(relative + absolute) including static html pages and not resources